src/l500/l500-options.cpp

```diff
@@ -36,23 +37,59 @@ namespace librealsense
         return _range;
     }
 
+    float l500_hw_options::query_default( hwmon_response * response ) const
```

Comment:
Add argument `streaming_sleep_for_old_fw` default to `true`, then can remove special code in `query_default`